### PR TITLE
doc/radosgw: improve grammar - notifications.rst

### PR DESCRIPTION
**Pull request by zdover23**
This commit makes the changes suggested by C. Mitchell in https://github.com/ceph/ceph/pull/48486.

Signed-off-by: Zac Dover <zac.dover@gmail.com>

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

File: doc/radosgw/notifications.rst [hunk 1]
```diff
@@ -95,20 +95,25 @@ Topics
...
 
 Create a Topic
 ``````````````
 
-This will create a new topic. The topic should be provided with push endpoint parameters that would be used later
-when a notification is created.
-Upon a successful request, the response will include the topic ARN that could be later used to reference this topic in the notification request.
-To update a topic, use the same command used for topic creation, with the topic name of an existing topic and different endpoint values.
+This creates a new topic. Provide the topic with push endpoint parameters,
+which will be used later when a notification is created. Upon a successful
```
Comment (colemitchell) [inline doc/radosgw/notifications.rst:106]: @zdover23 The sentence beginning with _Upon_ needs to be rewritten at some point.

**Review by anthonyeleven — APPROVED** [review]

File: doc/radosgw/notifications.rst (hunk 1 above)
Comment (zdover23) [reply doc/radosgw/notifications.rst:106]: @colemitchell I'll read up on this and figure out a better way to word it.

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/48494 - Quincy backport
https://github.com/ceph/ceph/pull/48495 - Pacific backport

File: doc/radosgw/notifications.rst (hunk 1 above)
Comment (zdover23) [reply doc/radosgw/notifications.rst:106]: @colemitchell: https://github.com/ceph/ceph/pull/48498 improves this sentence. However, the English throughout notifications.rst is confusing. We should rewrite it. I estimate that the rewriting of notifications.rst (and the insertion of unselectable prompts into the command lines) will take seven to nine hours and seven to ten distinct pull requests.

**Pull request merged**